launch/tier4_perception_launch/launch/object_recognition/detection/detector/lidar_dnn_detector.launch.xml

```diff
@@ -3,7 +3,7 @@
   <!-- Lidar parameters -->
   <arg name="lidar_detection_model" default="centerpoint" description="options: `transfusion`, `centerpoint`, `apollo`, `clustering`"/>
   <!-- Lidar detector centerpoint parameters -->
-  <arg name="centerpoint_model_name" default="centerpoint_tiny" description="options: `centerpoint`, `centerpoint_tiny` or `centerpoint_sigma`"/>
+  <arg name="centerpoint_model_name" default="centerpoint_tiny" description="options: `centerpoint`, `centerpoint_tiny` or `centerpoint_sigma` P.S. The _sigma model only has ml package version."/>
```

Comment:
While we appreciate you pointing out to us the state of the sigma model, leaving P.S. in files is not a good practice. I will notify the maintainers of this model, but in the meantime remove this comment